prisma/seed/forms/checkinformPO.ts

```diff
@@ -0,0 +1,89 @@
+import { prisma } from "../prisma-client";
+import { FormTitles } from "../../../src/global/constants/formTitles";
+
+export const populateCheckinFormPO = async () => {
+    // Sprint - checkin form (Product owner)
+    await prisma.form.create({
+        data: {
+            formType: {
+                connect: {
+                    name: "voyage member",
+                },
+            },
+            title: FormTitles.sprintCheckinPO,
+            description:
+                "Agile/Scrum is an important part of a Voyage and we'd like some feedback on how you are following ir and how your Product Owner and Scrum Master are helping your team.",
+            questions: {
+                create: [
+                    {
+                        order: 1,
+                        inputType: {
+                            connect: {
+                                name: "checkbox",
+                            },
+                        },
+                        text: "In what are your Product Owner's strengths?",
+                        answerRequired: true,
```

Comment:
I think this should be false according to the form (there is no red * after the question)

Could you also please go through the base check in form to see if the `answerRequired` matches the google form? I think some of them are wrong. Thanks!

Reply:
I will double-check it today!

Reply:
Today, I checked the google form and had make some changes to the checkin forms. Also at the top of the checkin form for PO and SM, there is a question at the top of that seems like it was a trigger for the forms. I didn't know where to put them so I added them to the top on the forms for each. let me know if that was the right move or should they be add to the basic checkin form or be their own form.

Reply:
@timDeHof we won't need those questions, as we will be able to tell if an SM/PO (or both) is in the team, without having to ask the user. Team composition is in the database

Reply:
ok...I removed those questions and updated the order of the questions...